meshtastic/node_runtime/channel_export_runtime.py

```
@@ -0,0 +1,223 @@
+            local_config = getattr(self._node, "localConfig", None)
+            if local_config is not None:
+                config_snapshot.CopyFrom(local_config)
+            return config_snapshot
+        with node_db_lock:
+            local_config = getattr(self._node, "localConfig", None)
+            if local_config is not None:
+                config_snapshot.CopyFrom(local_config)
+        return config_snapshot
+
+    def _build_and_encode_channel_set(
+        self, channels_snapshot: list[channel_pb2.Channel], include_all: bool
```

Ruff (0.15.7)

[warning] Boolean-typed positional argument in function definition

(FBT001)